assets/js/util.js

```diff
@@ -162,6 +165,8 @@ $(document).ready(function () {
         return false;
     });
 
+    $('#unobtrusiveWarning').addClass('hide');
```

Comment:
Ah, nevermind. The diff is being strange. Why don't you just add `hide` to the HTML?